cmd/completion.go

```diff
@@ -100,11 +100,27 @@ var powershellCompletionCmd = &cobra.Command{
 	Short: "Generate Powershell completion",
 	Long: `To load completions:
 
-We don't really know how Powershell works. 🙈
+You will need PowerShell >= version 5.0 to make this work, in order to use PSReadLine.
 
-This doc has a little more context: https://github.com/spf13/cobra/blob/master/powershell_completions.md
+If you don't have a PowerShell profile, or would like to make a new one:
 
-Want to improve this help text? Check out this issue: https://github.com/Netflix/weep/issues/17
+Create a $PROFILE file if needed
+
+$ if(!(Test-Path -Path $PROFILE)) {
+	New-Item -ItemType File -Path $PROFILE -Force
+}
+
+If you want to use an existing PowerShell profile:
+
+Open your profile with an editor (notepad used for example)
+
+$ notepad $PROFILE
+
+Add the navigable menu of the available options when you hit Tab
+
+$ Set-PSReadlineKeyHandler -Key Tab -Function MenuComplete
+
+You will need to start a new shell for this setup to take effect.
```

Comment:
From reading [the docs](https://github.com/spf13/cobra/blob/master/shell_completions.md#powershell-completions), it looks like users will still need to write the completion data to a file then source it from `$PROFILE`. Can you include that in these instructions?

Reply:
Added in the steps to write completion data to a file for sourcing from $PROFILE